micropub.php

```diff
@@ -552,26 +581,26 @@ public static function generate_post_content( $args ) {
 			$val = $props[ $prop ][0];
 			if ( $val ) {
 				$lines[] = '<p>' . $verbs[ $prop ] .
-						   ' <a class="u-' . $prop . '" href="' .
-						   $val . '">' . $val . '</a>.</p>';
+				  ' <a class="u-' . $prop . '" href="' .
+				  $val . '">' . $val . '</a>.</p>';
 			}
 		}
 
 		if ( isset( $props['rsvp'] ) ) {
 			$lines[] = '<p>RSVPs <data class="p-rsvp" value="' . $props['rsvp'][0] .
-				'">' . $props['rsvp'][0] . '</data>.</p>';
+			  '">' . $props['rsvp'][0] . '</data>.</p>';
 		}
 
 		// event
-		if ( static::$input['type'] == array( 'h-event' ) ) {
+		if ( array( 'h-event' ) === static::$input['type'] ) {
```

Comment:
it seems so silly that wordpress wants both yoda order _and_ `===`. i get that yoda order prevents accidental assignment, but that's not possible with `===`. meh.